render/vnode.js

```diff
@@ -1,7 +1,7 @@
 "use strict"
 
 function Vnode(tag, key, attrs, children, text, dom) {
-	return {tag: tag, key: key, attrs: attrs, children: children, text: text, dom: dom, domSize: undefined, state: undefined, events: undefined, instance: undefined, skip: false}
+	return {tag: tag, key: key, attrs: attrs, children: children, text: text, dom: dom, domSize: undefined, state: undefined, _state: undefined, events: undefined, instance: undefined, skip: false}
```

Comment:
@pygy What I meant was splitting the vnodes into a few specific object types:

- Text/Trusted: `{tag: "#"/"<", text}`
- Fragments: `{tag: "[", children}`
- Component/Node: `{tag, attrs, children, key, dom, state, ...privateState}`